Detectors/CTP/simulation/src/Digits2Raw.cxx

```diff
@@ -82,7 +84,7 @@ void Digits2Raw::processDigits(const std::string& fileDigitsName)
     std::vector<gbtword80_t> hbfIR;
     std::vector<gbtword80_t> hbfTC;
     for (auto const& ctpdig : CTPDigits) {
-      LOG(debug) << ctpdig.intRecord.bc << " bc all orbit " << ctpdig.intRecord.orbit;
+      LOG(info) << ctpdig.intRecord.bc << " bc all orbit " << ctpdig.intRecord.orbit;
```

Comment:
isn't this too verbose?

Reply:
I am still debugging, I comment/info->debug before removing draft pr.